ixwebsocket/IXWebSocket.cpp

```diff
@@ -13,16 +13,21 @@
 
 namespace
 {
-    uint64_t calculateRetryWaitMilliseconds(uint64_t retry_count)
+    uint64_t calculateRetryWaitMilliseconds(uint32_t retry_count)
     {
-        // This will overflow quite fast for large value of retry_count
```

Comment:
Can you keep that comment about overflow for the next programmer who come here ?

Reply:
No overflow now possible as I'm checking before: `if (retry_count <= 6)`

Reply:
Sure, but we should explain why we have this kind of odd magic value (6). Normally we wouldn't have to do that and would just use the pow(2) thing.